docs/reference/migration/migrate_8_0/mappings.asciidoc

```diff
@@ -9,33 +9,56 @@
 
 // end::notable-breaking-changes[]
 
-[float]
-==== Limiting the number of completion contexts
 
+.The maximum number of completion contexts per field is now 10.
+[%collapsible]
+====
+*Details* +
 The number of completion contexts within a single completion field
 has been limited to 10.
 
-[float]
-==== Removal of types
+*Impact* +
+To avoid errors, use no more than 10 completion contexts within a single
```

Comment:
As above (if you go that route):

> Use a maximum of 10 completion contexts in a completion field. Specifying more than 10 completions context will throw an error.

Same applies to the other impact sections.